packages/gas-fee-controller/src/GasFeeController.ts

```diff
@@ -589,6 +595,16 @@ export class GasFeeController extends StaticIntervalPollingController<
       this.currentChainId = newChainId;
     }
   }
+
+  #getAPIEndpoints() {
```

Comment:
Does this need to be an instance method? Can it be a function that lives outside the class instead?

Reply:
Absolutely not, there were more logic inside that function on first iterations, but now they are removed. 
Fixed in https://github.com/MetaMask/core/pull/4068/commits/0eda0b6752028fbda3f91d6fc20fea878f4ab43e